src/ui-kit/Divider/Divider.module.scss

```
@@ -0,0 +1,9 @@
+.divider {
+  border-bottom: rem(1px) solid $grey-medium;
```

```suggestion
  border-bottom: 1px solid $grey-medium;
```

quelque soit la taille du texte de l'utilisateur, tu souhaites que ta bordure soit de 1px, le cas échéant on se retrouverai probablement avec des sous-pixels, ce qui peut causer un effet un peu 'flou' sur certains navigateurs

---

la fonction rem vas rendre 1px.
Toutes les tailles doivent passé par rem() c'est un contrainte du linter.